Content.Server/Chat/Systems/ChatSystem.cs

```diff
@@ -214,11 +213,20 @@ public void TrySendInGameICMessage(
             _chatManager.EnsurePlayer(player.UserId).AddEntity(GetNetEntity(source));
         }
 
-        if (desiredType == InGameICChatType.Speak && message.StartsWith(LocalPrefix))
+        if (desiredType == InGameICChatType.Speak)
         {
-            // prevent radios and remove prefix.
-            checkRadioPrefix = false;
-            message = message[1..];
+            if (message.StartsWith(LocalPrefix))
+            {
+                // prevent radios and remove prefix.
+                checkRadioPrefix = false;
+                message = message[1..];
+            }
```

Comment:
Leave this as is and check for `desiredType == InGameICChatType.Speak && HasComp<HushedComponent>(source)` separately

Reply:
I put it in here since 228 needs to happen after the prefix removal but I could denote such in a comment if necessary.

Reply:
couldn't you just use the message with the removed prefix anyways and put the HasComp check after that if statement?

Reply:
Done